google_compute_engine/metadata_scripts/script_retriever.py

```diff
@@ -50,6 +50,14 @@ def _DownloadGsUrl(self, url, dest_dir):
     Returns:
       string, the path to the file storing the metadata script.
     """
+    try:
+       subprocess.check_call(
+         ['which', 'gsutil'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    except subprocess.CalledProcessError:
+      self.logger.warning(
+        'gsutil is not installed, cannot download items from Google Storage')
```

Comment:
Sorry - I think my attempt at giving a suggestion rendered incorrectly. Mind indenting by 4 for line continuations?